<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Piero Proietti</author>
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>400</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_snapshot_dir">
      <property name="text">
       <string>Nest</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditSnapshotDir"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_snashot_prefix">
      <property name="text">
       <string>Prefix</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditSnapshotPrefix"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_snapshot_basename">
      <property name="text">
       <string>Basename</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditSnapshotBasename"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_user_opt">
      <property name="text">
       <string>Live user name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditUserOpt"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_user_opt_passwd">
      <property name="text">
       <string>Live user password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditUserOptPasswd">
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_root_passwd">
      <property name="text">
       <string>Live root password</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditRootPasswd">
      <property name="echoMode">
       <enum>QLineEdit::PasswordEchoOnEdit</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="checkBoxMakeMd5sum">
      <property name="text">
       <string>md5sum</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBoxMakeIsohybrid">
      <property name="text">
       <string>isohybrid</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
